sound/soc/intel/boards/bxt_wm8804.c

```diff
@@ -225,7 +225,7 @@ static int bxt_wm8804_probe(struct platform_device *pdev)
 	int ret_val = 0;
 	int i;
 
-	mach = (&pdev->dev)->platform_data;
+	mach = pdev->dev.platform_data;
```

Comment:
would you mind putting this in a separate patch, this machine driver is not upstream so it'll require me or Ranjani to split it later. Thanks!

Reply:
@plbossart havent we made such changes to machine drivers, in a separate patch per machine driver before?

Reply:
I prefer one patch per machine driver when we change functionality so that bisections are easier but here it's fine.